Gemfile

```diff
@@ -141,7 +141,6 @@ group :development, :test do
   gem "bullet", "~> 6.1" # help to kill N+1 queries and unused eager loading
   gem "capybara", "~> 3.35.3" # Capybara is an integration testing tool for rack based web applications
   gem "cypress-rails", "~> 0.5.2" # For end to end tests (E2E)
-  gem "debug", "~> 0.2" # Debugging functionality for Ruby
```

Comment:
I wonder if this didn't work because it's not pointing to the 1.0 branch line: https://rubygems.org/gems/debug/versions/1.0.0.beta8

Reply:
Yes, probably. I'm not sure how I came up with that version 🤔 I'll try the 1.0 beta release now.

Reply:
This seems to have done the trick, thanks 🎉